Core/Core/Data/Model/Data_Dashboard.swift

```diff
@@ -43,57 +59,235 @@ public extension DataLayer {
+        public let courseUpdates: String
+        public let courseHandouts: String
+        public let discussionURL: String
+        public let videoOutline: String?
+        public let isSelfPaced: Bool
+
+        enum CodingKeys: String, CodingKey {
+            case id
+            case name
+            case number
+            case org
+            case start
+            case startDisplay = "start_display"
+            case startType = "start_type"
+            case end
+            case dynamicUpgradeDeadline = "dynamic_upgrade_deadline"
+            case subscriptionID = "subscription_id"
+            case coursewareAccess = "courseware_access"
+            case media
+            case courseImage = "course_image"
+            case courseAbout = "course_about"
+            case courseSharingUtmParameters = "course_sharing_utm_parameters"
+            case courseUpdates = "course_updates"
+            case courseHandouts = "course_handouts"
+            case discussionURL = "discussion_url"
+            case videoOutline = "video_outline"
+            case isSelfPaced = "is_self_paced"
+        }
+
+        public init(
+            id: String,
+            name: String,
+            number: String,
+            org: String,
+            start: String?,
+            startDisplay: String,
+            startType: StartType,
+            end: String?,
+            dynamicUpgradeDeadline: String?,
+            subscriptionID: String,
+            coursewareAccess: CoursewareAccess,
+            media: Media,
+            courseImage: String,
+            courseAbout: String,
+            courseSharingUtmParameters: CourseSharingUtmParameters,
+            courseUpdates: String,
+            courseHandouts: String,
+            discussionURL: String,
+            videoOutline: String?,
+            isSelfPaced: Bool
+        ) {
+            self.id = id
+            self.name = name
+            self.number = number
+            self.org = org
+            self.start = start
+            self.startDisplay = startDisplay
+            self.startType = startType
+            self.end = end
+            self.dynamicUpgradeDeadline = dynamicUpgradeDeadline
+            self.subscriptionID = subscriptionID
+            self.coursewareAccess = coursewareAccess
+            self.media = media
+            self.courseImage = courseImage
+            self.courseAbout = courseAbout
+            self.courseSharingUtmParameters = courseSharingUtmParameters
+            self.courseUpdates = courseUpdates
+            self.courseHandouts = courseHandouts
+            self.discussionURL = discussionURL
+            self.videoOutline = videoOutline
+            self.isSelfPaced = isSelfPaced
+        }
+    }
+
+    enum Name: String, Codable {
+        case courseImage = "Course Image"
+    }
+
+    enum Org: String, Codable {
+        case organization = "Organization"
+        case univerity = "Univerity"
```

Comment:
[curious] is this a typo that got missed? or is this a known edge case being handled this way?

Reply:
Not sure, need to check 🤔

Reply:
Fun fact, this enum is not used. 
I don't know how it ended up there, but I've removed it from the code.